<launch>
  <!-- Base Args -->
  <arg name="sim" default="false" doc="Whether to run in Aikido-sim only" />
  <arg name="version" default="2" doc="Kinova Arm Version (JACO = 2, Kortex = 3)" />
  <arg name="use_forque" value="true" />
  <arg name="use_apriltag_calib" value="false" />

  <!-- Feeding Args -->
  <arg name="autoRestart" default="false" doc="Whether to restart behavior tree at end without command line input." />
  <arg name="treeFile" default="feeding.xml" doc="BT root XML, from package://ada_feeding/trees" />

  <!-- Libada Config -->
  <include file="$(find libada)/launch/libada.launch" pass_all_args="true" />
  <!-- Libada Overrides -->
  <rosparam file="$(find ada_feeding)/config/gen$(arg version)_ada_config.yaml" command="load" ns="adaConf" />

  <!-- Launch Feeding Binary -->
  <node name="feeding"
          pkg="ada_feeding"
          type="feeding"
          output="screen"
          required="true">
    <remap from="~watchdog" to="/watchdog" />
    <remap from="~food_detector" to="/food_detector/marker_array" />
    <remap from="~face_detector" to="/face_detector/marker_array" />
    <remap from="~food_request" to="/alexa_msgs" />
    <param name="sim" value="$(arg sim)" />
    <param name="autoRestart" value="$(arg autoRestart)" />
    <param name="treeFile" value="$(find ada_feeding)/trees/$(arg treeFile)" />
    <rosparam file="$(find ada_feeding)/config/gen$(arg version)_feeding_config.yaml" command="load" />
  </node>

  <!-- green blue red-->
  <node pkg="tf2_ros" type="static_transform_publisher" name="camera_link_broadcaster"
      args="0.0185 0.065 0.034 0.50080661  0.49902445  0.50068027 -0.49948635 end_effector_link camera_link" />

  <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="camera_link_broadcaster" -->
  <!--    args="0.014 0.065 0.034 0.50080661  0.49902445  0.50068027 -0.49948635 end_effector_link camera_link" />     SAVED RAJAT VALUES -->

  <node pkg="tf2_ros" type="static_transform_publisher" name="bottom_camera_link_broadcaster"
    args="0.0122 -0.059  0.034 0.50080661  0.49902445  0.50068027 -0.49948635 end_effector_link bottom_camera_link" />

</launch>